### PR TITLE
parnters section

### PR DESCRIPTION
**Pull request by mlodyoskar**
This is how partners section looks like with some placeholder logos
image

### CODE REVIEW

File: src/components/main-page/Partners/Partners.module.scss [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+  color: var(--color-white);
+}
+
+.partnersIconsWrapper {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+  align-items: center;
+  gap: var(--space-xl) var(--space-3xl);
+  margin: var(--space-xl) 0 var(--space-2xl) 0;
```
Comment (marcol13) [inline src/components/main-page/Partners/Partners.module.scss:22]: Alternatively:
```suggestion
  margin: var(--space-xl) 0 var(--space-2xl);
```

File: src/components/main-page/Partners/Partners.module.scss [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+  max-width: 800px;
+}
+
+.partnerIcon {
+  transition: transform ease-in-out 0.25s;
+  transform: scale(1);
+  max-width: 128px;
+  height: 64px;
```
Comment (marcol13) [inline src/components/main-page/Partners/Partners.module.scss:30]: What do you think about export `max-width` and `height` in variables.css?

File: src/components/main-page/Partners/Partners.module.scss [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+}
+
+.partnerIconLink {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+
+  &:hover {
+    .partnerIcon {
+      transform: scale(1.125);
```
Comment (marcol13) [inline src/components/main-page/Partners/Partners.module.scss:40]: And if we would like to export `max-width` and `height` variables, maybe we should also create variable with `scale` value?

File: src/components/main-page/Partners/Partners.tsx [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+  }) as Partners[];
+
+  return (
+    <Section variant="white" id="partners" className={styles.section}>
+      <Heading className={styles.heading} withAccent>
+        {t("partners.title")}
+      </Heading>
+      <div className={styles.container}>
+        {partners.map(({ name, companies }) => {
+          return (
```
Comment (marcol13) [inline src/components/main-page/Partners/Partners.tsx:30]: We don't need `return` here - but it's minor thing - if you think that it's more readable with `return` you can leave it as it is 😀

File: src/components/main-page/Partners/Partners.tsx [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+            <div className={styles.container} key={name}>
+              <Badge color="secondary" className={styles.badge}>
+                {name}
+              </Badge>
+              <div className={styles.partnersIconsWrapper}>
+                {companies.map(({ name, logo, url }) => {
+                  return (
```
Comment (marcol13) [inline src/components/main-page/Partners/Partners.tsx:37]: Same situation

File: src/components/main-page/Partners/Partners.module.scss [hunk 6]
```diff
@@ -0,0 +1,43 @@
+.heading {
+  margin-bottom: var(--title-spacing);
```
Comment (marcol13) [inline src/components/main-page/Partners/Partners.module.scss:2]: It should be:
```suggestion
  margin-bottom: var(--font-size-2xl);
```
? Or new variable with `48px` as in Figma mockup

**Review by marcol13 — COMMENTED** [review]
Some minor suggestions from me 😉

File: src/components/main-page/Partners/Partners.tsx (hunk 4 above)
Comment (mlodyoskar) [reply src/components/main-page/Partners/Partners.tsx:30]: Yeah, I really like this explicit return, so I'd like to leave it as it is 😁

File: src/components/main-page/Partners/Partners.module.scss (hunk 6 above)
Comment (mlodyoskar) [reply src/components/main-page/Partners/Partners.module.scss:2]: That's for spotting it! Actually I was copy-pasting it from another section so I will fix it there too 😁

**Review by marcol13 — APPROVED** [review]

**Pull request merged**